index.rst

```diff
@@ -1,59 +1,374 @@
+   So even if a ``TAGGED`` collection is used, the public ``CHAINED`` collection would contain both that collection and the input ``CALIBRATION`` collection as children.
+
+.. note::
+
+   These public ``CHAINED`` collections essentially mimic Gen2's "parent link" mechanism, which provides at best approximate coarse-grained provenance information about which datasets were used as inputs when producing others.
+   The Gen3 repository will eventually be extended to include fine-grained, exact provenance - essentially a serialization of the directed acyclic graph (DAG) that describes the processing.
+   Whether queries against that DAG are fast enough to allow this more rigorous provenance information to be used as a type of collection (replacing some usage of ``TAGGED`` and ``CHAINED`` collections) remains to be seen, however.
+   It is also worth noting that in general the full DAG does not maintain the usual collection invariant of having only one dataset with a particular dataset type and data ID (e.g. two calexps with the same data ID, from two differently-configured runs, could each contribute to different, non-conflicting coadd patches in downstream runs).
+
+.. _collections-developer-processing-outputs:
+
+Developer processing outputs
+----------------------------
+
+Processing initiated by DM developers that are intended primarily for personal or small-group use must start with ``u/<user>`` (e.g. ``u/jbosch``), and are strongly encouraged to start with ``u/<user>/<ticket>`` (e.g. ``u/jbosch/DM-56789``) whenever possible.
+Names and structure after this prefix are at user discretion, but we strongly recommend using a combination of ``CHAINED`` collections and ``RUN`` collections to distinguish between "inputs and outputs" collections and "output only" collections, as in :ref:`collections-shared-official-processing-outputs`.
+The ``pipetask`` tool will automatically take care of this if the ``--output`` option is used with or instead of the ``--output-run`` option.
+
+.. note::
+   **TODO**: It's unclear whether BPS supports this currently, but it should be easy to at least support it under the condition that the ``RUN`` collection be given explicitly as well, instead of generated automatically by appending a timestamp.
+
+
+.. _collections-calibration-production:
+
+Calibration production
+----------------------
+
+Calibration production runs intended for broad use (i.e. outputs will be at least candidates for membership in the recommended calibration collection for this instrument) should output to collections with names that start with ``<instrument>/calib/<ticket>/``.
+Those produced for experimental or development purposes should start with ``u/<user>/<ticket>/``.
+
+In either case, the ``RUN`` collections that hold output datasets directly will usually require another disambiguating term, mapping roughly to the expected validity range epoch.
```

Comment:
I believe that the disambiguating term used for the RUN collections should simply be an estimated validStart value.  New calibrations (with updated validStart/validEnd values) will be necessary to track the time evolution of the camera-telescope system.  With the verification system currently in development, we should have metrics illustrating the quality of the currently certified calibrations, allowing us to track this evolution.  This means the new validStart date should not be completely unknown prior to calibration production.  This naming convention avoids the need to calculate the output collection from the input data (a problem with hashing), and should provide a human readable collection name.
I don't think a full range is helpful here, as calibrations are expected to be valid until replaced.  When generating the new calibration, we will likely always know the start date we want to use, but rarely the end date.

Reply:
The only qualm I have about using `validStart` as the final disambiguating term is that (because this term is essentially frozen into a dataset's identity forever), it is conceivable that a dataset could have a different actual `validStart` in one or more CALIBRATION collection, and hence users will have to be careful about identical or near-identical values actually representing different things in different contexts, and especially queries for that datasets that show a now-misleading RUN collection name.  I don't think that's a sufficient reason to drop the idea, but I wanted to make sure you were aware of the wrinkle.

Reply:
I've included this in the text and removed the TODO.